shared/run_diff.go

```diff
@@ -424,12 +425,23 @@ func runDiffFromSearchResponse(before, after TestRun, scDiff SearchResponse) (Ru
 		}
 	}
 
+	var renames map[string]string
+	if aeAPI.IsFeatureEnabled("diffRenames") {
```

Comment:
Ideally we should reuse this piece of code (it's exactly the same as in `GetRunsDiff`). Maybe next time.